lms/djangoapps/instructor/tests/test_api.py

```diff
@@ -2519,6 +2519,9 @@ def test_get_students_features(self):
         Test that some minimum of information is formatted
         correctly in the response to get_students_features.
         """
+        for student in self.students:
+            student.profile.city = "Mos Eisley {}".format(student.id)
+            student.profile.save()
```

Comment:
In this test, I decided to have country use the default value (when nothing has been set). There are more complete tests of the helper method in https://github.com/edx/edx-platform/pull/11620/files#diff-4b0f77beb25859a66087d2500f498d5aL109